playwright/test/work.test.ts

```diff
@@ -7,21 +7,27 @@ import {
 import { Page } from 'playwright';
 
 const getWhereToFindItAndEncoreLink = async (page: Page) => {
```

Comment:
np; should probably align this name as well. Also returns unavailable banner so the name is just a bit meh. `getAllStates`? 😅  not better. Naming is hard.

Reply:
It's a bit better and less wordy so fine I think.